src/main/java/uk/gov/hmcts/reform/divorce/orchestration/tasks/util/TaskUtils.java

```
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.divorce.orchestration.tasks.util;
+
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.TaskContext;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.TaskException;
+
+import java.util.Map;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.CASE_ID_JSON_KEY;
+
+public class TaskUtils {
+
+    public static String getMandatoryPropertyValueAsString(Map<String, Object> propertiesMap, String key)
```

Just remove this code and let the code blow up. If the field is not there. Even if you want to do this. Move this to context

---

Not sure what you mean by "move to context". Can you explain?